### DISCUSSION TITLE
Github Copilot for Individuals

### DISCUSSION THREAD

**Discussion by ghost** [community/community#56845] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Hello,

I am a solo developer, who does commercially used projects and I am curious which data from my sensitive code is collected through copilot.
Do I need the business version for my code to be safe? I read discussion [#50830 ](https://github.com/orgs/community/discussions/50830) and there it's only written for copilot for business.

Thanks

**Comment by Akash1134** [comment, marked as answer]
Hi ,

### From Copilot for Individuals Privacy FAQ - [What data does Copilot for Individuals collect](https://github.com/features/copilot#:~:text=Privacy%20%E2%80%93%20Copilot%20for%20Individuals)?

GitHub Copilot for Individuals relies on file content and additional data to work. It collects data to provide the service, some of which is then retained for further analysis and product improvements. GitHub Copilot processes the following data for individual users:

#### User Engagement Data
When you use GitHub Copilot it will collect usage information about events generated when interacting with the IDE or editor. These events include user edit actions like if Suggestions are accepted or dismissed, and error and general usage data to identify metrics like latency and features engagement. This information may include personal data, such as pseudonymous identifiers.

#### Prompts
A Prompt is the bundle of contextual information the GitHub Copilot extension sends when a user is working on a file and pauses typing, or when the user opens the Copilot pane. Prompts are only transmitted in real-time unless you have enabled telemetry collection, in which case Prompts are retained.

#### Suggestions
A Suggestion is one or more lines of proposed text returned to the GitHub Copilot extension after a Prompt is received and processed by the AI-model. Suggestions are only transmitted in real-time unless you enabled telemetry collection, in which case Copilot retains Suggestions.

 --> We follow responsible practices in accordance with our [Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement) to ensure that your code snippets will not be used as suggested code for other users of GitHub Copilot.

**Reply by ghost** [reply]
I just bought the license via the organization, thanks! I need the Industry-leading privacy, thanks for the help
